sdk/communication/Azure.Communication.MediaComposition/README.md

```diff
@@ -122,8 +122,8 @@ var gridMediaCompositionResponse = await mediaCompositionClient.GetAsync(mediaCo
 
 ### Updating an existing media composition
 
-Note: Updating the `groupCallInput` and the `teamsMeetingInput` is currently not supported. A new media composition will need to be created if `groupCallInput` and the `teamsMeetingInput` needs to change.
-```C# Snippet:UpdateMediaComposition
+You can update the layout:
```

Comment:
@minnieliu Could you regenerate the /api file to make it easier to see the full SDK interface diff?

Reply:
Yup, I've just regenerated in latest commit!